### PR TITLE
Updated git bisect scripts to accept more arguments

### PR DESCRIPTION
**Pull request by wtrzas2**

### CODE REVIEW

File: auto-check-fix-commit/git-bisect-runner.sh [hunk 1]
```diff
@@ -1,26 +1,35 @@
 #!/bin/bash
 
-for arg in "$@"; do
-  shift
-  case "$arg" in
-  '--flaky') set -- "$@" '-f' ;;
-  '--fixed') set -- "$@" '-x' ;;
-  '--module') set -- "$@" '-m' ;;
-  '--test') set -- "$@" '-t' ;;
-  *) set -- "$@" "$arg" ;;
```
[the remark this answers is not in the document]
Comment (wtrzas2) [reply auto-check-fix-commit/git-bisect-runner.sh:10]: When I tried to run the command with the arguments of both mvn and nondex version like so:

```
for arg in "$@"; do
  shift
  case "$arg" in
  '--flaky') set -- "$@" '-f' ;;
  '--fixed') set -- "$@" '-x' ;;
  '--module') set -- "$@" '-m' ;;
  '--test') set -- "$@" '-t' ;;
  '--mvn-install') shift; mvn_options="$1" ;;
  '--nondex-version') shift; nondex_version="$1" ;;
  *) set -- "$@" "$arg" ;;
  esac
done
```
I was getting an error that the arguments were not being recognized properly. When I just added in an argument for mvn-install it did, but adding in -nondex-version it didn't.

File: auto-check-fix-commit/git-bisect-runner.sh [hunk 2]
```diff
@@ -32,4 +41,4 @@ git checkout $flaky_commit
 
 git bisect good
 
-git bisect run ./git-bisect-script.sh $test_module $test_case
+git bisect run ./git-bisect-script.sh $test_module $test_case "$mvn_options" "$nondex_version"
```
[the remark this answers is not in the document]
Comment (wtrzas2) [reply auto-check-fix-commit/git-bisect-runner.sh:44]: I just tested it with multiple options for `$mvn_options` and it is registering it correctly. I also switched the order as suggested of `$nondex_version` and `$mvn_options`.

File: auto-check-fix-commit/git-bisect-runner.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (wtrzas2) [reply auto-check-fix-commit/git-bisect-runner.sh:10]: I had first just tried "-Dspotless.skip". This is the command I ran:
```
nohup ./git-bisect-runner.sh --flaky 6861ce03a2e1aabc315e41bdaae71a1912639e69 --fixed 429621f810336240e694dbbbb9fd1fcfef203276 --module galleon-pack/layer-metadata-tests --test org.wildfly.feature.pack.layer.tests.opentelemetry.OpenTelemetryLayerMetaDataTestCase#testClassInApiCommonPackage --mvn-install "-Dspotless.skip" --nondex-version "2.1.7" &> git_bisect_output.log
```

output log showed:
```
nohup: ignoring input
./git-bisect-runner.sh: illegal option -- D
./git-bisect-runner.sh: illegal option -- s
./git-bisect-runner.sh: illegal option -- p
./git-bisect-runner.sh: illegal option -- o
Flaky Commit: 
Fixed Commit: 429621f810336240e694dbbbb9fd1fcfef203276
Test Module: galleon-pack/layer-metadata-tests
Test Case: less.skip
Maven Options: --nondex-version
NonDex Version: 6861ce03a2e1aabc315e41bdaae71a1912639e69
Error: Missing required argument(s).
```

**Comment by darko-marinov** [comment]
Okay, fine to accept this as is, and some future students can work on extending this further.

**Pull request merged**